src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-notification-list/rgw-bucket-notification-list.component.html

```diff
@@ -0,0 +1,37 @@
+  <fieldset>
+    <legend i18n>
+    Notification Configuration
+    <cd-help-text>
+      Create a notification configuration to trigger alerts for specific bucket events, such as object creation or transitions, based on prefixes or tags.
```

Comment:
the page is for listing and managing the notification which is not represented in the text. So let's rephrase that.
```suggestion
      Configure bucket notification to trigger alerts for specific events, such as object creation or transitions, based on prefixes or tags.
```